lib/src/io.dart

```diff
@@ -568,6 +568,16 @@ Future<bool> confirm(String message) {
   return _stdinLines.first.then(RegExp(r'^[yY]').hasMatch);
 }
 
+/// Reads a line from stdin stream.
+Future<String> readLine(String message) {
+  if (runningFromTest) {
+    log.message('$message:');
+  } else {
+    stdout.write('$message: ');
```

Comment:
What happens if you always use log.message?

Reply:
I don't know 😅 I've just copied the code from `confirm(String message)` implementation.

Reply:
haha :) Then I guess the question is on us - probably the log.message prints a newline...